contrib/builder/deb/amd64/generate.sh

```diff
@@ -41,6 +41,15 @@ for version in "${versions[@]}"; do
 
 	echo >> "$version/Dockerfile"
 
+	if [ "$distro" = "debian" ]; then
+		cat >> "$version/Dockerfile" <<-EOF
```

Comment:
If you add single quotes around `EOF` here, you can drop the `\` on the `$` below, ie:
```bash
cat >> "$version/Dockerfile" <<-'EOF'
...
EOF
```